docs/advanced/test-command.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 1
+description: Test networking between Charon dependant services
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Network tests
+
+:::caution
+The charon test command is in alpha state and should be used with caution on testnets only.
```

Comment:
Is this really needed? In the end the test command never even interacts with real keys, its purpose is to be used before even creating a DV.

Reply:
I see your point, though running test commands doesn't really do any harm.

Maybe we can reword it mentioning something along the lines of "beta software, YMMV"?

Reply:
Changed it to:
`The charon test command is in alpha state and is still in development. It can not do any harm, but there is no guarantee it is stable and working as expected.`